public/css/module.less

```diff
@@ -70,8 +70,58 @@
 }
 
 .cert-donut {
+  @segment-color-0: #FF595E;
+  @segment-color-1: #FFCA3A;
+  @segment-color-2: #8AC926;
+  @segment-color-3: #1982C4;
+  @segment-color-4: #6A4C93;
```

Comment:
This way it's scoped to `.cert-donut`. Please put them into the module's global scope.

Reply:
✅